From 7c782700a82e6f8b1cb665075e14eaf9d4f70867 Mon Sep 17 00:00:00 2001
From: Fabian Deutsch <fabiand@fedoraproject.org>
Date: Thu, 25 Jun 2015 11:17:59 +0200
Subject: [PATCH] Revert "lvm: Don't activate LVs with activationskip set"

This reverts commit cfa365a32d47388c8476064b23d7d4684f1e591b.

The logic in commit cfa365a was added to prevent (odl) lvms from
activating snapshots which should not be activated.
Newer lvms however do this automatically (not enabling an LV if the
the 'k' attribute set), thus we can revert the previous commit.
---
 modules.d/90lvm/lvm_scan.sh | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/modules.d/90lvm/lvm_scan.sh b/modules.d/90lvm/lvm_scan.sh
index ce46e14..7495644 100755
--- a/modules.d/90lvm/lvm_scan.sh
+++ b/modules.d/90lvm/lvm_scan.sh
@@ -106,10 +106,6 @@ if [ -n "$LVS" ] ; then
     info "Scanning devices $lvmdevs for LVM logical volumes $LVS"
     lvm lvscan --ignorelockingfailure 2>&1 | vinfo
     for LV in $LVS; do
-        if [ "x$(lvm lvs --noheadings --select "lv_attr =~ k" $LV | wc -l)" = "x0" ]; then
-            info "Skipping activation of '$LV' because activationskip is set."
-            continue
-        fi
         if [ -z "$sysinit" ]; then
             lvm lvchange --yes -ay --ignorelockingfailure $nopoll --ignoremonitoring $LV 2>&1 | vinfo
         else
-- 
2.4.5

